.env.example

```diff
@@ -64,6 +64,7 @@ APP_EMAIL_NEW_USERS_NOTIFICATION=
 
 # Ability to disable signups on your instance.
 # Can be true or false. Default to false.
+# If this is set to true, signup related tests with fail
```

Comment:
I don't understand why adding this line?

Reply:
This is the default in `.env.example`. With this default set, the full test suite will fail locally and for new devs (like myself) it's not obvious what would cause these to fail. This helps new devs as they are debugging test failures.

Reply:
But this is not normal.
We could define a global value in `phpunit.xml` so it's possible to define any value in the `.env` file.
Tests must set the appropriate value in the test, see here: https://github.com/monicahq/monica/blob/0ae575f4a5b8571e0042ca8553e3948e23019687/tests/Feature/InstanceTest.php#L40

So I don't think this comment is relevant actually.

Reply:
But this is not normal.
We could define a global value in `phpunit.xml` so it's possible to define any value in the `.env` file.
Tests must set the appropriate value in the test, see here: https://github.com/monicahq/monica/blob/0ae575f4a5b8571e0042ca8553e3948e23019687/tests/Feature/InstanceTest.php#L40

So I don't think this comment is relevant actually.

Reply:
Ah, got it. `config` is missing from some of the registration tests. I'll fix this in another PR and remove this line.